framework/src/media/audio/audio_manager.h

```diff
@@ -153,11 +153,70 @@ audio_manager_result_t set_audio_stream_in(uint8_t channels, uint32_t sample_rat
  ****************************************************************************/
 audio_manager_result_t set_audio_stream_out(uint8_t channels, uint32_t sample_rate, uint8_t format);
 
+/****************************************************************************
+ * Name: get_input_frame_count
+ *
+ * Description:
+ *   Get the frame size of the pcm buffer in the active input audio device.
+ *
+ * Returned Value:
+ *   On success, the size of the pcm buffer for input streams. Otherwise, 0.
+ ****************************************************************************/
 uint32_t get_input_frame_count(void);
```

Comment:
how to validate between zero input frame count and error. Because in both the cases it returns '0'.

Reply:
The zero input frame count is also regarded as an error case. (Please see pcm_xxx functions in tinyalsa.c)
And to reduce unnecessary function calls to pcm_xxx, I added the zero checking logics.